armi/operators/settingsValidation.py

```diff
@@ -198,7 +199,13 @@ def run(self, cs=None):
                     self.__class__.__name__
                 )
             )
+
         else:
+            # preserve original settings file in case a corrective query is resolved
+            if any(query.isCorrective() for query in self.queries):
+                tempFilePath = "{}_temp.yaml".format(self.cs.path.split(".yaml")[0])
+                shutil.copyfile(self.cs.path, tempFilePath)
```

Comment:
just do shutil.copy instead of copy file. copy has the better beahvior i think 
![image](https://user-images.githubusercontent.com/50242044/235238751-b1bf7658-faf8-4314-92dc-29473c825839.png)

Reply:
Also, why can't this code be moved down to the other code. is there a reason the temp file is necessary? cant we just check if the _old exists and copy it if it does not?